GiEnJul/Controllers/RecipientController.cs

```diff
@@ -0,0 +1,77 @@
+﻿using GiEnJul.Features;
+using Microsoft.AspNetCore.Mvc;
+using Serilog;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+
+namespace GiEnJul.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class RecipientController : ControllerBase
+    {
+        private readonly IRecipientRepository _recipientRepository;
+        private readonly IPersonRepository _personRepository;
+        private readonly ILogger _log;
+
+        public RecipientController(IRecipientRepository recipientRepository, IPersonRepository personRepository, ILogger log)
+        {
+            _recipientRepository = recipientRepository;
+            _personRepository = personRepository;
+            _log = log;
+        }
+
+        [HttpPost]
+        public async Task<ActionResult<Entities.Recipient>> PostAsync([FromBody] Models.Recipient recipient)
+        {
+            _log.Debug("Adding recipient object: {@recipient}", recipient);
+            var recipientId = recipient.RowKey;
+            List<Entities.Person> addedPeople = new List<Entities.Person>();
+
+            for (var i = 0; i < recipient.FamilyMembers.Count; i++)
```

Comment:
I suggest using `foreach(var person in recipient.FamilyMembers)` for readability